js/MVP/containers/DocumentContainer.tsx

```diff
@@ -15,11 +14,19 @@ const DocumentWrapperStyled = styled.div`
   margin-top: 32px;
 `;
 
-const DocumentContainer = ({ document, lestDato, brevUuid, className, children }) => {
+interface DocumentContainerProps {
+  document: BrevDocument[];
+  className?: string;
+  children?: React.ReactNode;
+  lestDato?: string | undefined;
+  brevUuid: string;
+}
+
+const DocumentContainer = ({ document, lestDato, brevUuid, className, children }: DocumentContainerProps) => {
   const mutation = useMutateBrevLest();
 
   useEffect(() => {
-    if (brevUuid && !lestDato && !mutation.isLoading) {
+    if (!lestDato && mutation.isSuccess) {
```

Comment:
Det der vil vel ikke funke? !isLoading er der for å unngå å fyre flere events dersom mutation allerede er i gang. (holder på å mutere og venter på API-svar). Men dersom du sjekker isSuccess vil den vel aldri kjøre? Fordi den ikke blir isSuccess før etter mutation er kjørt

Reply:
Godt poeng! Her har jeg tenkt feil